root/user/ReportUser.js

```diff
@@ -62,7 +62,7 @@ const reportReasonOptions = {
     },
     {
       label: N_l(
-        'Editor has violated some other part of our Code of Conduct'
+        'Editor has violated some other part of our Code of Conduct',
```

Comment:
Coding style change in https://github.com/metabrainz/musicbrainz-server/pull/951/commits/87f85b374f4ad97d2b28cb03fdb626f7f8480c50 is unrelated to MBS-8922.